### PR TITLE
Added SwiftUI redacted tutorial

### PR DESCRIPTION
**Pull request by tunds**
## Checklist

* [x] The links are in the correct format: ``[Title of the Article](link), by [@author/`s Twitter username](link-for-twitter)``
* [x] I added my self to the credits with my GitHub account: ``[GitHub account](link-to-github)``
* [x] The link is freely available for everyone to read without the need to pay or to create a free account
* [x] I placed the link at the bottom of its category.
* [x] I acknowledge that there's another curation step and that merging this pull request doesn't automatically guarantee the submitted article will be in the newsletter. See [the contributing guides](https://github.com/iOS-Goodies/iOS-Goodies/blob/master/CONTRIBUTING.md#contributing)

## Optional

* [x] Article is not more than 2 weeks old
* [x] Article wasn't submitted before

### CODE REVIEW

File: Issues/Week377.md [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
 
-* 
+*
 
 **Videos**
 
-* 
+* [Create a placeholder loading screen with SwiftUI Redacted (SwiftUI Tutorial, Xcode 12, SwiftUI 2.0)](https://youtu.be/cfwEt__pnvA), by [@tundsdev](https://twitter.com/tundsdev)
 
 **Credits**
```
Comment (dkhamsing) [inline Issues/Week377.md:25]: Thx @tunds! Pls add yourself to the credits section after zntfdr

Comment (tunds) [reply Issues/Week377.md:25]: @dkhamsing Thanks I will do, can this be a link to my twitter?

Comment (tunds) [reply Issues/Week377.md:25]: @dkhamsing This has been done now, thanks for the heads up

**Comment by dkhamsing** [comment]
@tunds thx!

**Pull request merged**